NuKeeper.Update/Selection/IFilterSettings.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Text.RegularExpressions;
+
+namespace NuKeeper.Update.Selection
+{
+    public interface IFilterSettings
```

Comment:
Another "interface to DTO" for no reason, and there are others.  How did it come about? I would revert and start again with a simpler change that doesn't include this antipattern. I don't want that on master.